JenkinsfileBuildTrigger

```diff
@@ -0,0 +1,82 @@
+#!groovy
+
+
+def libraries = ['ci-pipeline':              ['master', 'https://github.com/CentOS-PaaS-SIG/ci-pipeline.git'],
+                 'upstream-fedora-pipeline': ['master', 'https://github.com/CentOS-PaaS-SIG/upstream-fedora-pipeline.git']]
+
+libraries.each { name, repo ->
+    library identifier: "${name}@${repo[0]}",
+            retriever: modernSCM([$class: 'GitSCMSource',
+                                  remote: repo[1]])
+
+}
+
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '500', daysToKeepStr: '', numToKeepStr: '500')),
+                parameters(
+                        [
+                                string(description: 'fedora-fedmsg', defaultValue: '{}', name: 'CI_MESSAGE')
+                        ]
+                ),
+                pipelineTriggers(
+                        [
+                                [$class: 'CIBuildTrigger', checks: [[expectedValue: '1', field: 'new']], providerName: 'fedora-fedmsg', selector: 'topic = "org.fedoraproject.prod.buildsys.build.state.change"']]
+                )
+        ]
+)
+
+def TRIGGER_RETRY_COUNT = 3
+def stepName = null
+def targetRelease = null
+
+node() {
+
+    ciMetrics.prefix = 'upstream-fedora-pipeline'
+    packagepipelineUtils.cimetrics = ciMetrics
+
+    try {
+        packagepipelineUtils.ciPipeline {
+            stepName = 'upstream-fedora-pipeline-build-trigger'
+            stage(stepName) {
+
+                packagepipelineUtils.timedPipelineStep(stepName: stepName, debug: true) {
+
+                    print "CI_MESSAGE"
+                    print CI_MESSAGE
+                    pipelineUtils.flattenJSON('fed', env.CI_MESSAGE)
+                    packagepipelineUtils.repoFromRequest()
+                    pipelineUtils.updateBuildDisplayAndDescription()
```

Comment:
updateBuildDisplayandDescription is the same thing as setBuildDisplayandDescription without the else. So, I think setBuildDisplayAndDescription is better in all cases. I have actually removed the update one from our other work. Don't have to change it, just my thoughts